config/routes.rb

```diff
@@ -3,7 +3,13 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   namespace :api, defaults: { format: :json } do
     resources :clients, :only => [:show, :create, :update, :destroy]
+    resources :tasks, :only => [:show, :create, :destroy]
```

Comment:
In terms of RESTful route conventions, the `task/show` route may be inappropriately named. This route is getting the tasks owned by some user so I think the route that makes the most sense is: `users/[:id]/tasks`. Translating that to plain english would sound like: "For user with id [:id], get me their tasks".

We can talk more about this later.